README.md

```diff
@@ -1134,6 +1134,8 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
   * [BurntSushi/rust-csv](https://github.com/BurntSushi/rust-csv) — A fast and flexible CSV reader and writer, with support for Serde [![build badge](https://api.travis-ci.org/BurntSushi/rust-csv.svg?branch=master)](https://travis-ci.org/BurntSushi/rust-csv)
 * EDN
   * [naomijub/edn-rs](https://github.com/naomijub/edn-rs) [[edn-rs](https://crates.io/crates/edn-rs)] — crate to parse and emit EDN format into Rust types. [![Build Status]( https://api.travis-ci.org/naomijub/edn-rs.svg?branch=master)](https://travis-ci.org/naomijub/edn-rs)
+* ENV
+  * [softprops/envy](https://github.com/softprops/envy) - deserialize env vars into typesafe structs [![Main](https://github.com/softprops/envy/actions/workflows/main.yml/badge.svg)](https://github.com/softprops/envy/actions/workflows/main.yml)
```

Comment:
Not sure if the order is correct

Not sure if the position it's correct

# ✅ More than 50 stars

https://github.com/softprops/envy/stargazers

![image](https://user-images.githubusercontent.com/36519478/212178662-f5e5fcdc-8318-4477-9669-44dc379e57dd.png)

# ✅ More than 2K downloads

https://crates.io/crates/envy

![image](https://user-images.githubusercontent.com/36519478/212178589-cc0596b0-2796-4494-a58b-2cea1c103f89.png)